runtime/vm/VMAccess.cpp

```diff
@@ -301,31 +301,33 @@ void   internalAcquireVMAccessNoMutexWithMask(J9VMThread * vmThread, UDATA haltM
 	}
 	Assert_VM_mustNotHaveVMAccess(vmThread);
 
-	if((vmThread->publicFlags & (J9_PUBLIC_FLAGS_JNI_CRITICAL_ACCESS | J9_PUBLIC_FLAGS_HALT_THREAD_EXCLUSIVE)) == (J9_PUBLIC_FLAGS_JNI_CRITICAL_ACCESS | J9_PUBLIC_FLAGS_HALT_THREAD_EXCLUSIVE)) {
+	if((vmThread->publicFlags & J9_PUBLIC_FLAGS_JNI_CRITICAL_ACCESS) == J9_PUBLIC_FLAGS_JNI_CRITICAL_ACCESS) {
 		/* In a critical region and about to block acquiring VM access. */
 		U_64 timeNow;
```

Comment:
This is a c++ file - can you move the definition of timeNow to its first use?

Reply:
Also, since this line is being changed, it would be good to use the J9_ARE_ANY_BITS_SET macro (or ALL_BITS if you want to be pedantic).

Reply:
re: Dan's comment, I believe he means to declare and initialize the variable at the same time.